plugins/BEdita/Core/src/Filesystem/Thumbnail.php

```diff
@@ -106,6 +106,16 @@ public static function get(Stream $stream, $options = 'default'): array
                 'acceptable' => false,
             ];
         }
+
+        if ($stream->mime_type === 'image/svg+xml') {
+            $ready = true;
+            $url = $stream->get('url');
+            $id = $stream->object_id;
+            $uuid = $stream->uuid;
+
+            return compact('ready', 'url', 'id', 'uuid');
```

Comment:
`id` and `uuid` are set in `MediaController`
```suggestion
            return [
                'ready' => true,
                'url' => $stream->get('url'),
            ];
```